include/seqan/file/file_page.h

```diff
@@ -1305,7 +1305,7 @@ firstToEnd(PageChain<TPageFrame> &pageChain)
         // Function is a functor which is called with a PageFrame object,
         // that is dirty or not READY (in an IO transfer)
         template <class Function>
-        inline int mru(Function Func_, unsigned maxLevel = PRIORITY_LEVELS - 1)
+        inline unsigned long mru(Function Func_, unsigned maxLevel = PRIORITY_LEVELS - 1)
         {
```

Comment:
why did you add unsigned here? The function actually can return -1 !! Also what types does PageLRUList acutally cover?